packages/apps/composer-app/src/main.tsx

```diff
@@ -139,11 +139,13 @@ const main = async () => {
       isDeck ? DeckMeta : LayoutMeta,
       NavTreeMeta,
       SettingsMeta,
-      HelpMeta,
       StatusBarMeta,
 
-      // Data integrations
+      // Shell and help
```

Comment:
I'm not sure I understand the significance of this group

Reply:
Help now needs shell (so it knows when it’s open), but shell was part of the group below it and help is not a data persistence plugin… I’m amenable to other ideas if you have any.

Reply:
does help need to come after client in order to get access to shell?

Reply:
Yes exactly

Reply:
why exactly? does help not render inside all of the contexts regardless?

Reply:
In order to get the shell display status, it needs `client`, which it can only get beneath client’s context provider.

Reply:
I see, help's logic is also in a context not in a component so the context order matters. We can fiddle with these later, this seem fine for now. Maybe just leave a clearer note stating that help needs to be after client for the context ordering.